addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -28,8 +28,12 @@ var ActivityRenderer = AbstractRenderer.extend({
             .removeClass('table-responsive')
             .empty();
 
-        if (this.state.data.activity_types.length === 0) {
-            this.$el.append($("<div>").text(_t("No schedulded activity")));
+        if (this.state.data.activity_types.length === 0) { 
+            this.$el.append(QWeb.render('GraphView.error', {
```

Comment:
this doesn't seem right... You should rename the template to something like 'View.helper' or 'View.noContentHelper'

Reply:
This view is also used to display errors (invalid data), so it isn't a noContentHelper either.

Reply:
````
<div t-name="GraphView.error" class="o_view_nocontent" role="alert">
    <div class="o_nocontent_help">
        <p class="o_view_nocontent_empty_folder"><t t-esc="title"/></p>
        <p><t t-esc="description"/></p>
    </div>
</div>
````
I think that a component should not be named according to usecases for which it is used, but rather according what it actually is. According to the html it produces, it is a no content helper.